src/MapperAxROM.cpp

```diff
@@ -0,0 +1,91 @@
+#include "MapperAxROM.h"
+#include "Log.h"
+
+namespace sn
+{
+
+    MapperAxROM::MapperAxROM(Cartridge &cart, std::function<void(void)> mirroring_cb) :
+    Mapper(cart, Mapper::AxROM),
+    m_mirroringCallback(mirroring_cb),
+    m_mirroring(OneScreenLower)   
+   {
+        if (cart.getROM().size() >= 0x8000)
+        {
+            LOG(Info) << "Using PRG-ROM OK" << std::endl;
+
+        }
+
+        if (cart.getVROM().size() <= 0x2000)
+        {
+
+            m_characterRAM.resize(0x2000);
+            m_ppubus.m_RAM.resize(0x400);
+            std::cout << "Uses Character RAM OK" << std::endl;
+
+            for (int i = 0; i < 0x2000; i++)
+            {
+                m_characterRAM[i] = 0;
+                //CHR RESET
+            }
+
+            for (int i = 0; i < 0x400; i++)
+            {
+                m_ppubus.m_RAM[i] = 0;
```

Comment:
I'm not sure what this is supposed to do here. `AxROM.h` creates a new ppubus that's not used by anything else?
Should this be a different array managed by the mapper or does this require something in the cartridge?